CommonUI/Sources/CommonUI/Buttons/ButtonStyle.swift

```diff
@@ -25,3 +25,16 @@ public struct LightGreenButton: ButtonStyle {
             .clipShape(Capsule())
     }
 }
+
+#Preview(body: {
```

Comment:
You don't need `body` specify.  
#Preview {
       VStack {
       .....
       }
 }